package.json

```diff
@@ -6,16 +6,12 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "test": "npm run lint",
+    "test": "npm run lint && lerna run test",
```

Comment:
@jquense btw, we weren't running tests previously. i had to take out a bunch of dummy `test` scripts inside the repos, though ... the ones that didn't have tests defined (including, oddly enough, svg2c, which had fixtures but no tests)